hpx/util/coroutine/detail/context_windows_fibers.hpp

```diff
@@ -49,7 +51,7 @@
 #include <boost/atomic.hpp>
 
 #if defined(HPX_HAVE_SWAP_CONTEXT_EMULATION)
-extern "C" void switch_to_fiber(void* lpFiber) throw();
+extern "C" void HPX_EXPORT switch_to_fiber(void* lpFiber) throw();
```

Comment:
Probably not, but for some weird reason I was not able to make it work without this change :/ It should probably a separate bug-fix, but I think it does not harm if it is left as a part of this PR.